.github/workflows/e2e-tests.yml

```diff
@@ -13,15 +13,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        parallel-id: [0, 1, 2, 3]
+        parallel-id: [0, 1, 2, 3, 4, 5, 6, 7]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v5
         with:
           go-version-file: "go.mod"
       - run: mkdir -p artifacts
-      - run: make e2e-local E2E_TEST_CHUNK=${{ matrix.parallel-id }} E2E_TEST_NUM_CHUNKS=${{ strategy.job-total }} E2E_NODES=2 ARTIFACT_DIR=./artifacts/ SKIP='\[FLAKE\]'
```

Comment:
Why is `E2E_NODES` decreased from 2 to 1?

Reply:
Having tests run in parallel seems to cause flakes. It didn't happen before because each test process would provision and install its own cluster (i.e. have its own cluster to run the tests against == no noisy neighbor).